modules/ossm-updating-smcp.adoc

```diff
@@ -26,14 +26,14 @@ $ oc project istio-system
 +
 . Edit the `ServiceMeshControlPlane` file.
 +
-.. Run the following command to edit the control plane where `<istio-installation.yaml>` includes a full path to the file you edited:
+.. Run the following command to edit the {SMProductShortName} control plane where `<istio-installation.yaml>` includes a full path to the file you edited:
 +
 [source,terminal]
 ----
 $ oc edit -n istio-system -f <istio-installation.yaml>
 ----
 +
-. Execute the following command to see the status of the control plane installation.
+. Execute the following command to see the status of the {SMProductShortName} control plane installation.
```

Comment:
```suggestion
. Run the following command to see the status of the {SMProductShortName} control plane installation.
```
Red Hat prefers "run" or "enter" instead of "execute."  Update the other instances of this word in this PR similarly.

Reply:
Done

Reply:
Searched all files. Done.